model/riscv_sys_regs.sail

```diff
@@ -508,6 +512,16 @@ function clause write_CSR((0x302, value) if xlen == 32) = { medeleg = legalize_m
 function clause write_CSR((0x312, value) if xlen == 32) = { medeleg = legalize_medeleg(medeleg, value @ medeleg.bits[31 .. 0]); medeleg.bits[63 .. 32] }
 function clause write_CSR(0x303, value) = { mideleg = legalize_mideleg(mideleg, value); mideleg.bits }
 
+bitfield Jvt : xlenbits = {
+  base : xlen - 1 .. 6,
+  mode : 5 .. 0
+}
+register jvt : Jvt
+mapping clause csr_name_map = 0x017  <-> "jvt"
+function clause is_CSR_defined (0x017) = sys_enable_zcmt()
+function clause write_CSR(0x017, value) = { jvt.bits = (value >> 6 ) << 6; jvt.bits }
```

Comment:
```suggestion
function clause write_CSR(0x017, value) = { jvt.bits = [value with 5 .. 0 = zeros()]; jvt.bits }
```

But it's probably better to make a `legalize_jvt()` function similar to the other `legalize` functions and then you can do something like:

```
function legalize_jvt(orig : Jvt, write : xlenbits) -> Jvt = {
  let write = Mk_Jvt(write);
  [orig with
    base = write.base,
  ]
}
```